sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@verax-attestation-registry/verax-sdk",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
Should we be changing it to 2.1.0 or 3.0.0?
As it can be a breaking change if someone is using nova in the code, it will stop working

Reply:
Alright, releasing v2.1.0 then! 🚀